pkgs/development/python-modules/mutagen/default.nix

```diff
@@ -25,10 +25,7 @@ buildPythonPackage rec {
     sha256 = "6397602efb3c2d7baebd2166ed85731ae1c1d475abca22090b7141ff5034b3e1";
   };
 
-  outputs = [
-    "doc"
-    "out"
-  ];
+  outputs = [ "out" "doc" ];
```

Comment:
Wow, ordering matters here? Call me surprised.

Reply:
Yeah, we where using the doc output. https://github.com/NixOS/nixpkgs/blob/master/doc/stdenv/multiple-output.chapter.md#using-a-split-package-sec-multiple-outputs-using-split-packages=

Reply:
Ah, so it's related to what gets picked for the consumer, understood. Thanks.